### ISSUE TITLE
Incorrect Typescript type for isSensorAvailable and onAttempt

### ISSUE THREAD

**Issue by CochiorasBogdan** [hieuvp/react-native-fingerprint-scanner#102]
Hello, the promise response type should be `Biometrics` instead of `string` at the following line:
[https://github.com/hieuvp/react-native-fingerprint-scanner/blob/9a698e2a137188a220d09c5259aa2de1e88944b7/index.d.ts#L90](url)

Alos the typing for `onAttempt` is incorrect since it can return an error ('UserCancel', 'UserFallback', etc.) like described in the documentation so the correct typing for: [https://github.com/hieuvp/react-native-fingerprint-scanner/blob/9a698e2a137188a220d09c5259aa2de1e88944b7/index.d.ts#L5](url) would be:
```
export type AuthenticateAndroid = { onAttempt: (error: FingerprintScannerError) => void };
```

**Comment by salomaoluiz** [comment]
Hi, I send a PR to fix this problem, now we just need wait the accept.

**Comment by phillbaker** [comment]
Fixed in #103
